apparmor.d/profiles-s-z/wechat-appimage

```diff
@@ -0,0 +1,95 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2025 EricLin
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{name} = wechat-appimage
+@{lib_dirs} = /opt/wechat-appimage/
+@{config_dirs} = @{user_config_dirs}/@{name}
+@{cache_dirs} = @{user_cache_dirs}/@{name}
+
+@{exec_path} = @{bin}/wechat @{lib_dirs}/wechat-appimage.Appimage /tmp/.mount_wechat??????/user/bin/wechat
+profile wechat-appimage @{exec_path} flags=(attach_disconnected) {
+  include <abstractions/base>
+  include <abstractions/attached/consoles>
+  include <abstractions/audio-client>
+  include <abstractions/common/electron>
+  include <abstractions/fontconfig-cache-read>
+  include <abstractions/app/bus>
+
+  capability dac_override,
+  capability dac_read_search,
+
+  network netlink raw,
+  network netlink dgram,
+  network inet stream,
+  network inet dgram,
+  network inet6 dgram,
+  network inet6 stream,
+
+  @{exec_path} r,
+
+  @{sh_path}  rix,
+  @{lib_dirs}/wechat-appimage.AppImage ix,
+  /tmp/.mount_wechat??????/AppRun ix,
+  @{bin}/mkdir ix,
+  @{bin}/gawk  rix,
+  @{bin}/lsblk rix,
+  @{bin}/ip rix,
+  @{bin}/xdg-user-dir rix,
+  @{tmp}/.mount_wechat??????/opt/wechat/{,**} ix,
+  @{tmp}/.mount_wechat??????/usr/bin/wechat ix,
+  @{open_path} rpx -> child-open-strict,
+
+  mount fstype=fuse.wechat-appimage.AppImage options=(ro nodev nosuid) wechat-appimage.AppImage -> @{tmp}/.mount_wechat??????/,
+
+  umount @{tmp}/.mount_wechat??????/,
+
+  @{bin}/fusermount{,3} ix -> fusermount,
+  @{bin}/dirname  rix -> fusermount,
+  @{bin}/readlink rix -> fusermount,
```

Comment:
That won't work this way. You need:
`@{bin}/fusermount{,3} Cx -> fusermount,`
And the other are simple `rix,`